recipes/h3-py/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "h3-py" %}
+{% set version = "3.4.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/h/h3/h3-{{ version }}.tar.gz
+  sha256: a292a5a63bcf535f0137141019f3defd21ad2fbd258dc8de587f921cf2c5b41d
+
+build:
+  number: 0
+  skip: True  # [win]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - h3 # TODO: how can I overwrite https://github.com/uber/h3-py/blob/master/setup.py to use the conda provided H3 https://github.com/conda-forge/h3-feedstock
```

Comment:
How can I overwrite to use https://github.com/conda-forge/h3-feedstock?

Reply:
You can patch the file. See https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#patches

Reply:
Btw, `h3` should be in `host`. (`build` should be empty for this package)

Reply:
Moved. Regarding patching:
the `.install.sh` file (https://github.com/uber/h3-py/blob/master/.install.sh) only contains the native code part, not python. So I assume this is already completely handled by the new conda-force h3 package.

I.e. I only need to patch https://github.com/uber/h3-py/blob/master/setup.py is this correct?

Reply:
But patching fails with:
```
ile "/opt/conda/lib/python3.7/site-packages/conda_build/source.py", line 518, in _guess_patch_strip_level
    histo = {i: 0 for i in range(maxlevel + 1)}
TypeError: unsupported operand type(s) for +: 'NoneType' and 'int'
```